climada/util/coordinates.py

```diff
@@ -122,13 +122,16 @@ def lon_normalize(lon, center=0.0):
     lon : np.array
         Longitudinal coordinates
     center : float, optional
-        Central longitude value to use instead of 0.
+        Central longitude value to use instead of 0. If None, the central longitude is determined
+        automatically.
```

Comment:
This is a very small, cheap change that might come in handy in the future (and is already useful for some of my usecases).